2024-04-22 13:43:35,645 INFO    MainThread:1407761 [internal.py:wandb_internal():90] W&B internal server running at pid: 1407761, started at: 2024-04-22 13:43:35.643922
2024-04-22 13:43:35,646 DEBUG   HandlerThread:1407761 [handler.py:handle_request():140] handle_request: run_start
2024-04-22 13:43:35,646 INFO    WriterThread:1407761 [datastore.py:open_for_write():75] open: /scratch/spantang/CS678-TPol/wandb/offline-run-20240422_134334-39d5g2wf/run-39d5g2wf.wandb
2024-04-22 13:43:35,648 DEBUG   HandlerThread:1407761 [meta.py:__init__():35] meta init
2024-04-22 13:43:35,648 DEBUG   HandlerThread:1407761 [meta.py:__init__():49] meta init done
2024-04-22 13:43:35,648 DEBUG   HandlerThread:1407761 [meta.py:probe():209] probe
2024-04-22 13:43:35,663 DEBUG   HandlerThread:1407761 [meta.py:_setup_git():199] setup git
2024-04-22 13:43:35,684 DEBUG   HandlerThread:1407761 [meta.py:_setup_git():206] setup git done
2024-04-22 13:43:35,684 DEBUG   HandlerThread:1407761 [meta.py:_save_pip():53] save pip
2024-04-22 13:43:35,704 DEBUG   HandlerThread:1407761 [meta.py:_save_pip():67] save pip done
2024-04-22 13:43:35,704 DEBUG   HandlerThread:1407761 [meta.py:_save_conda():74] save conda
2024-04-22 13:43:41,420 DEBUG   HandlerThread:1407761 [meta.py:_save_conda():84] save conda done
2024-04-22 13:43:41,421 DEBUG   HandlerThread:1407761 [meta.py:probe():247] probe done
2024-04-22 13:43:45,124 WARNING MainThread:1407761 [internal.py:wandb_internal():153] Internal process interrupt: 1
2024-04-22 13:43:45,251 WARNING MainThread:1407761 [internal.py:wandb_internal():153] Internal process interrupt: 2
2024-04-22 13:43:45,252 ERROR   MainThread:1407761 [internal.py:wandb_internal():156] Internal process interrupted.
2024-04-22 13:43:45,389 INFO    HandlerThread:1407761 [handler.py:finish():806] shutting down handler
2024-04-22 13:43:45,725 INFO    WriterThread:1407761 [datastore.py:close():279] close: /scratch/spantang/CS678-TPol/wandb/offline-run-20240422_134334-39d5g2wf/run-39d5g2wf.wandb
2024-04-22 13:43:45,729 INFO    SenderThread:1407761 [sender.py:finish():1114] shutting down sender
2024-04-22 13:43:45,734 INFO    MainThread:1407761 [internal.py:handle_exit():80] Internal process exited
